server/header_context_test.go

```diff
@@ -80,19 +84,29 @@ func TestMetadataHeadersExceptions(t *testing.T) {
 
 	h := &Handler{
 		orgName: "org",
-		envName: "env",
+		envName: "*",
 	}
 	h.targetHeader = "target"
-	header := map[string]string{"target": "target"}
+	header := map[string]string{
+		"target":          "target",
+		headerEnvironment: "test",
+	}
+
 	api, ac := h.decodeMetadataHeaders(header)
```

Comment:
nit: this `api` assignment is not ineffectual and should be `_`. golangci actually spotted this but had it filtered out.